components/user_settings/security/user_settings_security.test.tsx

```diff
@@ -125,7 +138,7 @@ describe('components/user_settings/display/UserSettingsDisplay', () => {
             preventDefault: jest.fn(),
         };
         wrapper.setState({authorizedApps: apps});
-        wrapper.instance().deauthorizeApp(event);
+        (wrapper.instance() as UserSettingsSecurity).deauthorizeApp(event);
```

Comment:
You need to update the calls `shallow` in this file to explicitly set the type to
`const wrapper: ShallowWrapper<any, any, UserSettingsSecurity> = shallow(...`.

With that you can even remove these type cases.

Reply:
I have written in this `const wrapper: any = shallow(<UserSettingsSecurity {...props}/>)` using this also I am not getting any errors  while checking types will that be okay or I need to change it to `const wrapper: ShallowWrapper<any, any, UserSettingsSecurity> `

Reply:
I'd prefer you use `ShallowWrapper<any, any, UserSettingsSecurity` but either way is fine.